sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2061,8 +2062,26 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
           if (ignore) {
             assert(aclEntries.size() == 0)
           } else {
-            assert(aclEntries.size() == 1)
+            assert(aclEntries.size() == 4)
```

Comment:
Technically, this test the following.
1. The acls are generated by the permission always.
2. The custom acl is survived without by overwritten by (1).

If the above is your intention, it would be great if you describe that into the PR description together.

Reply:
Make sense. Let me update the PR description.